modules/update-availability-faq.adoc

```diff
@@ -32,6 +32,8 @@ The primary purpose of the `eus` channel is to serve as a convenience for cluste
 
 * The only difference between releases on the `fast` and `stable` channels is that a release only appears on the `stable` channel after it has been on the `fast` channel for some time, which provides more time for new update risks to be discovered.
 
+* Releases that are available on the `fast` channel will always become available on the `stable channel` after this delay.
```

Comment:
Is the "will always" important? Usually we try to avoid "will"/future tense

```suggestion
* Releases that are available on the `fast` channel become available on the `stable channel` after this delay.
```